clientgui/DlgAdvPreferencesBase.cpp

```diff
@@ -1080,6 +1085,11 @@ bool CDlgAdvPreferencesBase::doesLocalPrefsFileExist() {
     return local_prefs_found;
 }
 
+// to make result available externally
+bool CDlgAdvPreferencesBase::isUsingPrivatePrefs() {
```

Comment:
I don't think I've heard them called private prefs before. Maybe `isUsingLocalPrefs` instead?

Reply:
You're right. Changed.